docs/source/snakemake/lifecycle.md

```diff
@@ -1,12 +1,12 @@
-# Lifecycle of a Snakemake Execution on Latch
+# Snakemake Execution Lifecycle
 
 Snakemake support is currently based on JIT (Just-In-Time) registration. This means that the workflow produced by `latch register` will register a second workflow, which will run the pipeline tasks. This is because the actual structure of the workflow cannot be specified until parameter values are provided.
 
 ### JIT Workflow
```

Comment:
It's hard to conceptualize how JIT workflow maps to Latch Console/ the workflow page. 

- Add a screenshot of what the JIT step looks like (e.g. only 1 task, the logs show that files are being downloaded, etc.)